packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx

```diff
@@ -228,10 +236,10 @@ export function EditForm({
   );
 }
 
-type AttributeType = z.infer<typeof attributeSchema>;
+type Attribute = z.infer<typeof attributeSchema>;
```

Comment:
Avoids confusion with AttributeType which is an enum representing Attribute['type']